core/tee/tee_svc_cryp.c

```diff
@@ -3277,6 +3277,9 @@ TEE_Result syscall_asymm_operate(unsigned long state,
 		}
 		break;
 
+#if defined(CFG_SECURE_KEY_SERVICES)
```

Comment:
I would prefer another `CFG_`. Maybe introduce `CFG_CRYPTO_RSASSA` to default enable in the related make script.